### ISSUE TITLE
Refactor: add tsd-files defined in tsd.json to gitignore

### ISSUE THREAD

**Issue by sclausen** [mgechev/angular2-seed#169]
Tracking them is IMHO unnecessary and bloats the git repo.

**Comment by ludohenin** [comment]
@sclausen this is because installing typings on npm preinstall is not stable considering we don't want to rely upon tsd being globaly installed.
Btw If you have a working solution (working on Travis as well) that'd be awesome :)

**Comment by sclausen** [comment]
Installing typescript definitions has to be done before compiling any typescript and you can't use the local tsd before installing it, so why not moving the tsd commands in the gulp-task to to the postinstall npm task?
``` json
    "postinstall": "tsd reinstall --clean; tsd link; tsd rebundle; gulp postinstall",
```
``` typescript
gulp.task('postinstall', done =>
  runSequence('clean',
              'npm',
              done));
```
now you would execute tsd commands after installing the node modules, but before executing any gulp scripts and you could add `tools/typings/tsd` to the `.gitignore`.

**Comment by ludohenin** [comment]
Is this (`"postinstall": "tsd reinstall --clean; tsd link; tsd rebundle; gulp postinstall",`) cross platform ?

**Comment by sclausen** [comment]
:frowning: I didn't thought there could be any compatibility problems. Unfortunately, I don't have multiple os on hand, so how could I test this?

**Comment by sclausen** [comment]
I tested it on travis-ci and it succeeds.

**Comment by ludohenin** [comment]
It works on OSX for me.
Anyone who can test on Windows ?

**Comment by svrcekmichal** [comment]
@ludohenin it works :) i prefer `&&` syntax, but it's running ok

**Comment by ludohenin** [comment]
Good to read!I also prefer `&&` but wasn't sure if it'd run across platforms.
@sclausen would you send a PR (single commit & with latest master) ?

**Comment by sclausen** [comment]
@ludohenin Here you go #171

**Comment by sclausen** [comment]
closed with #173
